src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,10 +157,27 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
+    std::uint32_t deleted_ledger_index = 0;
```

Comment:
you dont need to define this, you can directly use `output.deleted_ledger_index`, similar to `nodeBinary`